core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -110,7 +110,7 @@ THE SOFTWARE.
                 </st:include>
                 <td class="pane" align="center" valign="middle">
                   <j:if test="${e.hasStopPermission()}">
-                    <l:stopButton href="${rootURL}/${c.url}${url}/stop" confirm="${%confirm(exe.fullDisplayName)}" alt="${%terminate this build}"  />
+                    <l:stopButton href="${rootURL}/${c.url}${url}/stopBuild?runExtId=${h.urlEncode(exe.externalizableId)}" confirm="${%confirm(exe.fullDisplayName)}" alt="${%terminate this build}"  />
```

Comment:
I think that `h.urlEncode` expects a non null object. But if `exe` is not a `Run` that method is not defined on the object and I think jelly will then just pass in `null` causing a `NullPointerException`

Reply:
You're right. I've addressed that in db5d364, by making `h.urlEncode` tolerant to `null`. It will then return an empty string, which is, I think, more convenient than returning `null` for the intended use case (encoding URL query parameters).

Reply:
Oh, and you're right about jelly passing `null` for unknown properties, I tested it to be sure.